scapy/utils6.py

```diff
@@ -395,7 +359,7 @@ def in6_getLocalUniquePrefix():
     tod = struct.pack("!II", i,j)
     # TODO: Add some check regarding system address gathering
     from scapy.arch import get_if_raw_hwaddr
-    rawmac = get_if_raw_hwaddr(conf.iface6)[1]
+    rawmac = get_if_raw_hwaddr(conf.iface)[1]
```

Comment:
It allows the function to work without a working `conf.iface6`. It is silly anyway. I used `RandMac` instead as the RFC does not specify which interface must be used.